src/main/java/org/jenkinsci/plugins/durabletask/Controller.java

```diff
@@ -55,7 +69,7 @@
     /**
      * Checks whether the task has finished.
      * @param workspace the workspace in use
-     * @param launcher a way to start processes
+     * @param launcher a way to start processes (currently unused)
```

Comment:
The method is overridable, and the downstream implementation may actually use it but inherit Javadocs. So I would rather not touch it